PlayerKit/Classes/RegularPlayer.swift

```diff
@@ -125,7 +125,8 @@ open class RegularPlayer: NSObject, Player, ProvidesView
     {
         let cmTime = CMTimeMakeWithSeconds(time, Int32(NSEC_PER_SEC))
         
-        self.player.seek(to: cmTime, toleranceBefore: kCMTimeZero, toleranceAfter: kCMTimeZero)
+//        self.player.seek(to: cmTime, toleranceBefore: kCMTimeZero, toleranceAfter: kCMTimeZero)
+        self.player.seek(to: cmTime)
```

Comment:
I'm down with this change if this doesn't have any consequences. Let's remove the commented out code though.